mlrun/feature_store/api.py

```diff
@@ -195,18 +195,39 @@ def get_online_feature_service(
     """initialize and return online feature vector service api,
     returns :py:class:`~mlrun.feature_store.OnlineVectorService`
 
-    example::
+    There are two ways to use the function
+    1. As context manager
+        example::
+
+                with get_online_feature_service(vector_uri) as svc:
+                    resp = svc.get([{"ticker": "GOOG"}, {"ticker": "MSFT"}])
+                    print(resp)
+                    resp = svc.get([{"ticker": "AAPL"}], as_list=True)
+                    print(resp)
+
+            example with imputing::
+
+                with open get_online_feature_service(vector_uri, impute_policy={"*": "$mean", "amount": 0)) as svc:
+                    resp = svc.get([{"id": "C123487"}])
+    2. as simple function, note that in that option you need to close the session.
+        example::
+
+
+            svc = get_online_feature_service(vector_uri)
```

Comment:
Since we're giving here a good example, I suggest you also wrap the code with a `try` and use `finally` to close the service - the example code as-is is exactly what we want to avoid.